sdk/eventhub/Azure.Messaging.EventHubs/src/Primitives/CheckpointStore.cs

```diff
@@ -78,17 +79,24 @@ public abstract class CheckpointStore
         /// <param name="consumerGroup">The name of the consumer group the checkpoint is associated with.</param>
         /// <param name="partitionId">The identifier of the partition the checkpoint is for.</param>
         /// <param name="offset">The offset to associate with the checkpoint, intended as informational metadata. This will only be used from positioning if there is no value provided for <paramref name="sequenceNumber"/>.</param>
-        /// <param name="sequenceNumber">The sequence number to associate with the checkpoint, indicating that a processor should beging reading from the next event in the stream.</param>
+        /// <param name="sequenceNumber">The sequence number to associate with the checkpoint, indicating that a processor should begin reading from the next event in the stream.</param>
         /// <param name="cancellationToken">A <see cref="CancellationToken" /> instance to signal a request to cancel the operation.</param>
         ///
-        [EditorBrowsable(EditorBrowsableState.Never)]
         public virtual Task UpdateCheckpointAsync(string fullyQualifiedNamespace,
                                                    string eventHubName,
                                                    string consumerGroup,
                                                    string partitionId,
                                                    long offset,
                                                    long? sequenceNumber,
-                                                   CancellationToken cancellationToken) => UpdateCheckpointAsync(fullyQualifiedNamespace, eventHubName, consumerGroup, partitionId, null, new CheckpointPosition(sequenceNumber ?? long.MinValue, offset), cancellationToken);
+                                                   CancellationToken cancellationToken)
```

Comment:
Did we want to make this optional?  I think that was the outcome of discussions?

Reply:
I don't remember - I had thought the outcome was that they were not optional here because this is an abstract class

Reply:
I don't think it would be a breaking change since there's no overload that could confuse callers.   I'm slightly in favor but don't feel strongly.   I'll defer to you for the final decision.